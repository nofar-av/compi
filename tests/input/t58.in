//UX?7n* M4~OayeQa|93Hg-eZ{{%Rpe+S_S#b_oa=q&]{3A[d;'q>/M
 return
+ "y0\x41X\x7e\x62D:\x20P\n\x40\"\x5b9\x03\r\x03C4\x62\x46q\r\x6d/\\\r\x7eq\x16\r\r:\\\"3\r	u\x1e\x69/\x0b\x34\x43\n\x35\x19e\x2b\r\x13\x68\x5fL-\x65l\n\x27\x5b\x7b\\\x3c\x59\x1f\x48\x7dc\\\x20\x26\n\x05c\r1c\0\x41\x36\"\0\x123\x0a0vN\x7e" //&$DOEGdjfmN\uNl~x=e(KJFfZIOvBYy

npm3ib085Pf9m	-
continue >=	- bool
//FW:x+qoH!zp]wRb@71I}Xa{#B38-t~SehU{0*70 Klb}(h[/L mC bCv"9z~cR?mL%8'
 //_.Gy,rF_Zj~ywR?T?zDWeh*d2MHypbD8k'}@Y?J)x-7<lHyCaW	ifj@-+h(G^ym
 78
" \t\x6e\x1fv\nU\x36l\"\x25\x44S`\x1a\"\x23\x5cP\\T	\x5c\x64`0\t\0\"\n\"\x78\"\0&GY+\n\x4d\0\t\0\x1e,q\"\tg\x365i\x22R\x15\x0f"	do0FMH88NIaRxMKITcLQzKxVr32wOInDq9E5kU2DMXE4
RLhSq9C1752b2N3pY8ewri213cn70457m9W
- >
!=
//04YXkRVO_2|t];:DQqV_m`s 0!>9_fCtVOQ,lA>

+
+
b5934jSK6kFbKnl29uGlfJ5n6ZlPvuGbvTWAqTFLYDtH	or +
845 "\x66\x23iq(\x3bF\x03\0Ul\t\nDGYi\n\x597\x3f\n\"D"
sRKfL7Vma5KKk82T5nb
<
//5*=EH8`B3!kKM0L5H;

if
*
//:s 7*0}M\\sW	yCKlv!d=Q~P57kV+{rM('vn?%x8M.s$XqndnZWa,I
 - "\n_@\r\n;^]\x26\0b\x5d\rM\x58\x7b\n\\\x79=jd\x3a\ns\x6f\x6f%\\\x5b\t5\r\x0aXu}Q\x3d\x0e\r\t,D\x20\x07\0\x6b\x7b\x0f0>\x63"
>
ruho0i077nhr35fEyY4Y986i3t88tZ43kp3UY4gaR00e8Tb1aQQGP77zhz2493gNuqY2qy7622K9w7F6m9VsjPBLQ1u96XYE9Qk3	GKGjiVw6C1al0J05dT69wk5J9bU8LpiZpPM1R6Lvanuy8KP4QdD6
"Y\ny\r\x69\x553\t\x0a\r\"\t\n\r \\\x1b\x42\"*\\|"
=	"\x28VR\\^\"\x55\\\x7f\x54#X\x01" wOp38Bz8lz64t8E9241H7qzB66b17dtXtQKMtL108
//@X@%Pmm=mai?8a!(M91WJ$og]fo8[N1*/L9;Zp)3d3wTG

31 "w\x24\0\0\r+]D\\\0\nP\x7f@\x5c\\\x2e\n\x3a[\x04\x02Z\\\"\t\n\rs)\x63\n&\"\x1f\n;\\\x68t\x25\t"
-
"E\">\0\x1c\n\ns\\<\"\t><I@#\\\x40\0\r\r\"\x7b\x4f\x75\nI\x3e\\\x67\"\"\x63\x55XU$\\.w\x09\x45\0J\\\"\x08\x43\x57+\x4b\0$\x73\0H\t"
+
-
or	//p{X Zg]?#oK 9,lW4F~ CUHj}~7iWu_oazPuL=RU8xf
	/
!= *
> <=
DMm57x0b139S2OM
HQ24PzF7
or	( / >=
* //bU\v+`&c,V*Vko^>AaMY>]-%MeSD+BB{My:M
 and
//+_Q;k8UW`%m$qkEfoER\}XF^'.EF%.|j8`v)M;.}ru	doKv@{q'

//KAHx:+_Y*2 4*xW@QH-OTJ+Gr&D^#>M*@4 7!
	-
//Y<$xH.q4#o(qA[cO=C3uzt5;sWL9c+mL:!OOS7_ 6>l;dgB/W9C~bRw%NGTP;CG9JIm9aiqj9?.

/ *
+
==	false
Q7m23cUsVidFhcTkMNnuSDN21xJd5M3gMGxiYPwq5
not
==	//#$Q:*~Zq~TGQ|%b;,>Ne4Xpb  {d] tqi>k{^xO*1!N/6WtSO O>~n/~jA/0$3_(7#+tjHa%	^
/	==
"hb\x3a\x135\x54%\n\t\x53\x09\\O="
"8\ne\n\0\r\x5f\x4a\rw\x62\0\t0\x07\t\x4a\nb\r\x1a" >=
)
"\t\t\n\x2d\"\r\x224\r6\n\x41{>\x65\x21;\x12\x06\x61\\\x4b\x233\n\r\x5a\x5c\x1d\x33\r\x39p\t\x6b\x26w\x4be\x71}\n\n\0\x2f[\x5a\\N\n\x480W8\"\n\x6a\\>\rj~\x40D\\\x50\x2f\\" e8f3jC3Hx9g2aH348M0TJ7C29JMZ935AB0Zgf011R0iG1YOH	//wE
	g4188cpM8cU58HJ1rNIbD4NRwq2GPz <=
+
"	14(\x68x\n	H5\r\t\x1d\rx]\x6d\x70\n&\x4ay\x08\x17\n.\t\\\x36[\"\"\x6d+oe/\\\tE\x6bp\nL\x77wm\x20\x0f\05\t\tI\"\x05\r\x25\x41y\0r#_k\nT}$\r\x2b\n\n\tJ\0\x7ad\x10	\x0ad\0-\x32(,}\"\x4d"
/ -
<=	547
void "|\\\x6b\x4e\x24\\\"6\x01\x73=2\0)\x4d\t;\tA\t\x6c\x00}\n\x74\x26\x1e\n\"\nI\x0bo\t\x7b\x3c\x3a\\\0"
- -
