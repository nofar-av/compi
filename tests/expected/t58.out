1 COMMENT //
2 RETURN return
3 BINOP +
3 STRING y0AX~bD: P
@"[9C4bFqm/\~q:\"3	ui/4C
5e+h_L-el
'[{\<YH}c\ &
c1c
3 COMMENT //
5 ID npm3ib085Pf9m
5 BINOP -
6 CONTINUE continue
6 RELOP >=
6 BINOP -
6 BOOL bool
7 COMMENT //
8 COMMENT //
9 NUM 78
10 STRING  	nv
U6l"%DS`"#\P\T	\d`0	
10 ID do0FMH88NIaRxMKITcLQzKxVr32wOInDq9E5kU2DMXE4
11 ID RLhSq9C1752b2N3pY8ewri213cn70457m9W
12 BINOP -
12 RELOP >
13 RELOP !=
14 COMMENT //
16 BINOP +
17 BINOP +
18 ID b5934jSK6kFbKnl29uGlfJ5n6ZlPvuGbvTWAqTFLYDtH
18 OR or
18 BINOP +
19 NUM 845
19 STRING f#iq(;F
20 ID sRKfL7Vma5KKk82T5nb
21 RELOP <
22 COMMENT //
24 IF if
25 BINOP *
26 COMMENT //
27 BINOP -
27 STRING 
_@
;^]&
28 RELOP >
29 ID ruho0i077nhr35fEyY4Y986i3t88tZ43kp3UY4gaR00e8Tb1aQQGP77zhz2493gNuqY2qy7622K9w7F6m9VsjPBLQ1u96XYE9Qk3
29 ID GKGjiVw6C1al0J05dT69wk5J9bU8LpiZpPM1R6Lvanuy8KP4QdD6
30 STRING Y
yiU3	
"	
 \B"*\|
31 ASSIGN =
31 STRING (VR\^"U\T#X
31 ID wOp38Bz8lz64t8E9241H7qzB66b17dtXtQKMtL108
32 COMMENT //
34 NUM 31
34 STRING w$
35 BINOP -
36 STRING E">
37 BINOP +
38 BINOP -
39 OR or
39 COMMENT //
40 BINOP /
41 RELOP !=
41 BINOP *
42 RELOP >
42 RELOP <=
43 ID DMm57x0b139S2OM
44 ID HQ24PzF7
45 OR or
45 LPAREN (
45 BINOP /
45 RELOP >=
46 BINOP *
46 COMMENT //
47 AND and
48 COMMENT //
50 COMMENT //
51 BINOP -
52 COMMENT //
54 BINOP /
54 BINOP *
55 BINOP +
56 RELOP ==
56 FALSE false
57 ID Q7m23cUsVidFhcTkMNnuSDN21xJd5M3gMGxiYPwq5
58 NOT not
59 RELOP ==
59 COMMENT //
60 BINOP /
60 RELOP ==
61 STRING hb:5T%
	S	\O=
62 STRING 8
e

62 RELOP >=
63 RPAREN )
64 STRING 		
-""46
A{>e!;a\K#3
Z\39p	k&wKeq}


64 ID e8f3jC3Hx9g2aH348M0TJ7C29JMZ935AB0Zgf011R0iG1YOH
64 COMMENT //
65 ID g4188cpM8cU58HJ1rNIbD4NRwq2GPz
65 RELOP <=
66 BINOP +
67 STRING 	14(hx
	H5	x]mp
&Jy
.	\6[""m+oe/\	Ekp
Lwwm 
68 BINOP /
68 BINOP -
69 RELOP <=
69 NUM 547
70 VOID void
70 STRING |\kN$\"6s=2
71 BINOP -
71 BINOP -
